Semver/Ranges/Comparers/Npm/NpmRange.cs

```diff
@@ -0,0 +1,145 @@
+using System;
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using System.Linq;
+using System.Text;
+using System.Text.RegularExpressions;
+
+namespace Semver.Ranges.Comparers.Npm
+{
+    /// <summary>
+    /// A range of versions that can be checked against to see if a <see cref="SemVersion"/> is included.
+    /// Uses the same syntax as npm.
+    /// </summary>
+    public class NpmRange : IRange
```

Comment:
I think this should be directly in the `Semver.Ranges` namespace to avoid exposing `Comparers` namespace to users of the library since they don't see any actual comparers.